app/Http/Controllers/DashboardController.php

```diff
@@ -40,6 +41,34 @@ public function getExpenses()
     public function getExpenseSummary($expenses){
       //make an summary
       //return new
+        $summary = array();
+        $owed = DB::table('expenses AS e')
+            ->leftjoin('sharedexpenses AS se', 'e.id', '=', 'se.expense_id')
+            ->select('e.*', 'se.secondary_username', 'se.amount_owed')
+            ->where('e.owner_username', Auth::user()->username)
+            //->get();
+            ->sum('se.amount_owed');
+        $owing = DB::table('expenses AS e')
+            ->leftjoin('sharedexpenses AS se', 'e.id', '=', 'se.expense_id')
```

Comment:
Again if you're just calculating the total for shared expenses, you don't need to know who its owed to here, no join is necessary.